### ISSUE TITLE
Buggy Firefox Transitions

### ISSUE THREAD

**Issue by svanetten1976** [malsup/cycle2#663]
HI malsup,
   First of all, thanks for developing Cycle and Cycle2 in the first place. I've been using the original cycle for years for many different projects and applications, not just slideshows. Anyway, I've started to transition to using Cycle2 on some projects as of late, and I've noticed some really buggy behavior in newer versions of Firefox. I'm seeing this on Firefox 35 on a Mac, with jQuery 1.8.1 and the newest version of Cycle2 (2.1.6). Have yourself or anyone else noticed or reported this issue, and if so, is there a fix? Thanks, Steve

**Comment by stef157** [comment]
If you mean that the picture is cut and after x ms it apear correctly.. Then I've te same problem in cycle2..

**Comment by FlashJunior** [comment]
if you mean the image position is lightly out on transition:
checkout #403, #520 
add `.cycle-slide img{box-shadow:#fff 0 0 0;}` to your css

**Comment by svanetten1976** [comment]
Yep, that's the primary symptom I've got going on there, and its weird
cause its with Firefox only (which I usually find to be the best
browser). You have any idea what is causing the problem?
